### DISCUSSION TITLE
How to suppress for bundled file name comment by terser-webpack-plugin's extractComments

### DISCUSSION THREAD

**Discussion by hamilton-keisuke** [webpack/webpack#16649] [Q&A]
This is my sample repo.
https://github.com/hamilton-keisuke/terser-webpack-plugin-sample

I tried to bundle js by webpack with terser-webpack-plugin `extractComment: true`.
Webpack output such [license.txt](https://github.com/hamilton-keisuke/terser-webpack-plugin-sample/blob/master/dist/index.js.LICENSE.txt) .

I could not find below comment in `./src/js/index.js` , `./src/js/jquery.js` and `./node_modules/jquery/dist/jquery.js`.

```
/*! ./jquery.js */

/*! jquery */

/*!*************************!*\
  !*** ./src/js/index.js ***!
  \*************************/

/*!**************************!*\
  !*** ./src/js/jquery.js ***!
  \**************************/

/*!********************************************!*\
  !*** ./node_modules/jquery/dist/jquery.js ***!
  \********************************************/

```

Where such comments come from?
Is it possible to suppress outputting only such comments to the lisence.txt?

**Comment by alexander-akait** [comment]
Do you want to disable any comments in files? Please look at this https://github.com/webpack-contrib/terser-webpack-plugin#extractcomments and provide your configuration of the terser plugin, thank you
